pkgs/applications/networking/instant-messengers/telegram/tdesktop/default.nix

```
@@ -75,7 +75,7 @@ mkDerivation rec {
     repo = "tdesktop";
     rev = "v${version}";
     fetchSubmodules = true;
-    sha256 = "0hpcsraw3gx37wk3r88q51qf68ny0kb3kd6slnklwf22d1jqr3fn";
+    sha256 = "1397rsb1srzzfvbbqafhpf1jpx8iz13vj8y6gyq44gr9zq54iwcm";
```

```suggestion
    sha256 = "0sgfhicb81a5cn05f9zi7333kwh76qbnija1fyw0cxrrwap9d73m";
```
Sirs, this is `3.4.0`'s hash.